### ISSUE TITLE
Recruit more Tech Volunteers

### ISSUE THREAD

**Issue by richpomfret** [FreeUKGen/Coordination#333]
Build code team - next cycle of volunteers needed.

**Comment by Captainkirkdawson** [comment]
Are we going to use cloud9 as their development framework or an equivalent approach? The experience with the last 3 has shown our existing process is a waste of our time and resources.

**Comment by richpomfret** [comment]
Yes! We think we can use an Education package with Cloud9 (which is super cheap) and it should reduce the risk on our side in terms of wasting time/resources. Has the Cloud9 experience remained positive with continued use?

**Comment by Captainkirkdawson** [comment]
It has indeed been positive.

**Comment by richpomfret** [comment]
Includes a need now for a Visual Basic coder to take over Mike Fry's efforts (see Eric's email).
